### ISSUE TITLE
Documents should not include Schemas

### ISSUE THREAD

**Issue by danilobuerger** [dotansimha/graphql-code-generator#1229]
**Describe the bug**

```yaml
// codegen.yml

schema:
  - ./src/schema.gql
documents: ./src/**/*.gql
```
Errors out as the schema is included in the documents.

```
 ✔ Parse configuration
 ❯ Generate outputs
   ❯ Generate ./src/types.ts
     ✔ Load GraphQL schemas
     ✔ Load GraphQL documents
     ✖ Generate
       → Found x errors in your documents


  Found 1 error

   ✖ ./src/types.ts
    Found x errors.
    Operations and fragments were validated against the schema.
    Please fix following errors and run codegen again:

        ./src/schema.gql: 
          The xxx definition is not executable.
```

**To Reproduce**

Use above `codegen.yml` with a local schema.

**Expected behavior**

Documents should exclude any Schemas.

**Comment by MrDesjardins** [comment]
I am having the same error. I have .graphql file that contains types and query. For some reason, the tool output "The XXX definition is not executable". It happens when I specify the `schema` and `document` with the same blob to get my GraphQL schema from all my .graphql.

Avoiding the `document` entry works, but the TypeScript definition file is empty, which is not better.

**Comment by dotansimha** [comment]
Hi @danilobuerger @MrDesjardins ! Thank you for reporting this issue.
I think in this case, as a workaround, it might be easier for you to use a different way to specify the glob to your documents (such as `src/**/components/*.graphql` or similar, or use `something.query.graphql`). 
I'll take a look soon, maybe it's not too difficult to filter the irrelevant `DocumentNode`s from the found files).

**Comment by danilobuerger** [comment]
@dotansimha My workaround for now is naming queries `.gql` and schema `.graphql`. But this feels super weird.

**Comment by dotansimha** [comment]
@danilobuerger You can also use a suffix for your files instead of a different extension, it should work (`type.schema.graphql` and `data.query.graphql`).

**Comment by dotansimha** [comment]
we moved the logic of files loading to [`graphql-toolkit`](https://github.com/dotansimha/graphql-toolkit), and I fixed it there: https://github.com/dotansimha/graphql-code-generator/pull/1257 

@danilobuerger @MrDesjardins

**Comment by dotansimha** [comment]
Fixed in 0.16.1 :)

**Comment by danilobuerger** [comment]
Confirmed working. Thanks @dotansimha
